drivers/display/ssd16xx.c

```diff
@@ -217,6 +208,32 @@ static int ssd16xx_update_display(const struct device *dev)
 	return ssd16xx_write_cmd(dev, SSD16XX_CMD_MASTER_ACTIVATION, NULL, 0);
 }
 
+static int ssd16xx_blanking_off(const struct device *dev)
+{
+	struct ssd16xx_data *data = dev->data;
+
+	LOG_DBG("");
+
+	if (data->blanking_on) {
+		ssd16xx_busy_wait(dev);
+		data->blanking_on = false;
+		return ssd16xx_update_display(dev);
+	}
+
+	return 0;
+}
+
+static int ssd16xx_blanking_on(const struct device *dev)
+{
+	struct ssd16xx_data *data = dev->data;
+
+	LOG_DBG("");
+
+	data->blanking_on = true;
+
+	return 0;
```

Comment:
The whole point of the blanking state in this driver and the other EPD driver (GD7965) is to prevent screen refreshes while the ram is being updated. Forcing a refresh here would introduce unnecessary flickering just before a software updates the frame buffer.